about/xdev.md

```diff
@@ -1,479 +1,164 @@
-and open source software best practices through example.
+To achieve this, Xdev acts as navigators for NCAR scientists in the Pangeo sea.  As such, Xdev helps NCAR scientists adopt a common approach to enabling broad multidisciplinary collaboration through its education and development efforts.  Xdev’s _education_ efforts to teach scientists at NCAR (and beyond) Python and the Pangeo stack help build and strengthen the Pangeo community of technology _users_.   Meanwhile, Xdev’s _development_ efforts to collaborate with scientists on Pangeo technology development help teach open development and open source software best practices through example.
 
 
 ## The Xdev Manifesto
 
-Xdev is an NCAR entity, and therefore must satisfy NCAR priorities and
-strategic objectives.  Xdev is also commited to the _external_ Pangeo community,
-and therefore it behoves us to enumerate our _technical values_ to highlight how
-Xdev’s values align with Pangeo.
-
-1. We value **interactivity** over batch processing.  While batch processing
-   is still necessary for many things, we believe that the need for
-   curiosity-driven analysis is greater and that this should be the priority.
-
-2. We value **open source** over closed source solutions.  While some closed
-   source solutions may be better for specific needs, especially if a monetary
-   investment has already been made, we believe that open source solutions are
-   a better long term solution in general.
-
-3. We value **scalability**.  While serial, non-scalable solutions can be
-   reasonable algorithm prototypes, they are of limited value in the era of
-   Big Data.  We believe that scalability should be considered at all times.
-
-4. We value workflow **portability**.  While having solutions that only work
-   on NCAR systems can solve immediate problems, we want to reduce the burden
-   on scientists for having to learn different solutions on different
-   platforms, whether they be HPC or cloud.  Scientists should decide where to
-   do their analysis based on where the data is hosted, not based on where
-   they can run the specific code they want to run.
-
-5. We value **transparency** over opacity.  While some work for many reasons
-   cannot be made public, we choose to do all of our work in the open so that
-   others can learn from us, leverage our work, and work with us at any time.
-
-6. We value **reproducibility**.  While static, non-reproducible scientific
-   communication still has value as the traditional mechanism for exchanging
-   knowledge throughout scientific disciplines, we believe reproducible
-   mechanisms for publishing and sharing scientific knowledge are more
-   important and effective.
+Xdev is an NCAR entity, and therefore must satisfy NCAR priorities and strategic objectives.  Xdev is also commited to the _external_ Pangeo community, and therefore it behoves us to enumerate our _technical values_ to highlight how Xdev’s values align with Pangeo.
+
```

Comment:
```suggestion
Xdev is an NCAR entity, and therefore must satisfy NCAR priorities and strategic objectives.  Xdev is also committed to the _external_ Pangeo community, and therefore it behooves us to enumerate our _technical values_ to highlight how Xdev’s values align with Pangeo.

```

Reply:
Thanks!  I wrote this fast, so I'm sure there are more typos.  😄